### DISCUSSION TITLE
MeanDice ( monai.handlers.mean_dice) is not updating in multi-label segmentation task

### DISCUSSION THREAD

**Discussion by saruarlive** [Project-MONAI/MONAI#1457] [Q&A]
Hi, @Nic-Ma et al,

Could you kindly provide a simple toy example using the meandice handler in multi-label segmentation task?
I asked that before that the meandice score is not updating in the training log. It works while testing on random pred/target

![image](https://user-images.githubusercontent.com/24542592/104366646-fe10f680-5519-11eb-8cd6-629deee13ae0.png)

**Comment by Nic-Ma** [comment]
Hi @saruarlive ,

If you want to compute `MeanDice` in ignite program or MONAI engines, you can refer to this example:
https://github.com/Project-MONAI/tutorials/blob/master/modules/engines/unet_training_dict.py#L137
If you just want to compute `MeanDice` metrics in multi-labels task in regular PyTorch program, you can refer to:
https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/brats_segmentation_3d.ipynb

Thanks.
